### PR TITLE
Implements email success/failure messages with templates

### PR DESCRIPTION
**Pull request by knod**

### CODE REVIEW

**Comment by knod** [comment]
Maybe worth checking out https://docassemble.org/docs/recipes.html#python%20to%20javascript (which I was just given).

**Comment by knod** [comment]
Sorry, it seems I accidentally deleted something that was meant to be in there.

**Comment by knod** [comment]
Should be ready for review now.

**Comment by knod** [comment]
Something about uploading artifacts didn't work, which confuses me a bit.

File: docassemble/juvenilesealing/data/questions/document-download-screen.yml [hunk 1]
```diff
@@ -110,37 +82,43 @@ fields:
...
-          })
-        }
-      });  // ends url_action_call and callback
-    });
-
+         daHideSpinner();
+         if (response.status === 200) {
+           //console.log(obj_or_html)
+           //console.log(result_as_text)
+           //console.log(response)
```
Comment (niharikasingh) [inline docassemble/juvenilesealing/data/questions/document-download-screen.yml:112]: If we're not using these comments anymore, remove entirely?

File: docassemble/juvenilesealing/data/questions/document-download-screen.yml [hunk 2]
```diff
@@ -110,37 +82,43 @@ fields:
...
-    console.log(val('email_sent_prefix'))
-
+  
     $("#sendAttachmentEmail").click(function(){
-
-      let message = val('email_attempt_prefix') + val('atch_email_address') + ".";
+      daShowSpinner();
+      
+      var email_attempt_prefix = ${ json.dumps(str(email_attempt_prefix))}
+      var message = email_attempt_prefix + val('atch_email_address') + ".";
```
Comment (niharikasingh) [inline docassemble/juvenilesealing/data/questions/document-download-screen.yml:93]: This needs a space between the attempt_prefix text and the email address: `email_attempt_prefix + " " + val('atch_email_address') + "."`

**Pull request merged**